controller/main_controller.php

```diff
@@ -93,7 +93,7 @@ public function display($route)
 	* @param string $route The route name for a page
 	* @return object $entity The entity object
 	* @throws http_exception
-	* @access public
+	* @access protected
```

Comment:
@Nicofuma is there any reason not to have it... as in, is it worth all the work to remove it from everywhere?